Docker/build/WDQS-frontend/custom-config.json

```diff
@@ -15,7 +15,7 @@
       "urlShortener": "tinyurl"
     },
     "brand": {
-      "title": "$BRAND_TITLE",
+      "title": "${WDQS_NAME}",
```

Comment:
I agree $BRAND_TITLE isn't a great choice for the env var here, but would probably maybe require updating some documentation too if you search for $BRAND_TITLE in the repo it occurs a couple of times.

Reply:
I thought BRAND_TITLE was clear enough and find it clearer than WDQS_NAME. Note how the JSON calls it "brand title" as well, so there also is the consistency aspect.

Reply:
Yeah on a second glance I agree that consistency should have priority here, lets not do this change.

Reply:
The reason I changed this is to have all of the user adjustable settings in .env file start with WDQS_. While this brakes a bit of consistency within this template file, it increases the consistency for the user in the .env file, where they would have BRAND_TITLE next to WDQS_FRONTEND_HOST and WDQS_FRONTEND_PORT. Maybe this could be renamed to WDQS_FRONTEND_NAME though.

Reply:
Fair point, maybe a compromise could be `WDQS_FRONTEND_BRAND_TITLE`, just rolls off the tongue ;)

Reply:
Sounds good. Now that we found a compromise, the only piece of documentation that needs to be adjusted is the [README.md](https://github.com/wmde/wikibase-release-pipeline/blob/10f338e04ff796b59e638d2a5abcebaa8e0e6d7d/Docker/build/WDQS-frontend/README.md?plain=1#L10). I can add another commit to this PR, but not before next week.